tests/functional/spec/timeout_modal.spec.js

```diff
@@ -0,0 +1,47 @@
+import TimeoutInterstitialPage from "../generated_pages/timeout_modal/timeout-modal-interstitial.page";
```

Comment:
Need one to capture, the modal displaying an us not doing anything to ensure after the countdown, the modal is not displayed and that we have been redirected to the expected page.

Reply:
So I used the 5s session expiry schema for this, otherwise we would have to set timeout to more than 125 seconds.